frontend/packages/dev-console/integration-tests/features/monitoring/dashboard-display.feature

```diff
@@ -1,67 +1,59 @@
-        Scenario: Dashboard tab on the Monitoring page for particular workload: M-02-TC02
+        Scenario: Dashboard tab on the Observe page for particular workload: M-02-TC02
             Given user is at Add page
               And user has created workload "parks-test" with resource type "Deployment"
               And user opened the url of the workload "parks-test" in topology page
-              And user is on Monitoring page
+              And user is on Observe page
              When user clicks on "Dashboard" tab
               And user selects the workload "parks-test" from the dropdown
-             Then user will see the CPU Usage on Dashboard tab
-              And user will see the Memory Usage on Dashboard tab
-              And user will see Receive Bandwidth on Dashboard tab
-              And user will see Transmit Bandwidth on Dashboard tab
-              And user will see the Rate of Received Packets on Dashboard tab
-              And user will see the Rate of Transmitted Packets on Dashboard tab
-              And user will see the Rate of Received Packets Dropped on Dashboard tab
-              And user will see the Rate of Transmitted Packets Dropped on Dashboard tab
+             Then user will see the "Namespace (Workloads)" on Dashboard tab
+              And user will see the "Namespace (Pods)" on Dashboard tab
+              And user will see "Workloads" on Dashboard tab
+              And user will see "Pods" on Dashboard tab
 
 
         @regression
-        Scenario: Dashboard tab on the Monitoring page for all workloads: M-02-TC03
+        Scenario: Dashboard tab on the Observe page for all workloads: M-02-TC03
             Given user is at Add page
               And user has created workload "parks-test-1" with resource type "Deployment"
               And user opened the url of the workload "parks-test-1" in topology page
-              And user is on Monitoring page
+              And user is on Observe page
              When user clicks on "Dashboard" tab
              Then user will see the dropdown selected with All Workloads by default
-             Then user will see the CPU Usage on Dashboard tab
-              And user will see the Memory Usage on Dashboard tab
-              And user will see Receive Bandwidth on Dashboard tab
-              And user will see Transmit Bandwidth on Dashboard tab
-              And user will see the Rate of Received Packets on Dashboard tab
-              And user will see the Rate of Transmitted Packets on Dashboard tab
-              And user will see the Rate of Received Packets Dropped on Dashboard tab
-              And user will see the Rate of Transmitted Packets Dropped on Dashboard tab
+              And user will see the "Namespace (Workloads)" on Dashboard tab
+              And user will see the "Namespace (Pods)" on Dashboard tab
+              And user will see "Workloads" on Dashboard tab
+              And user will see "Pods" on Dashboard tab
```

Comment:
Now all workloads dropdown will be not visible upfront. And user will see these options in the dashboard dropdown.

Reply:
Updated